services/status/locales/i18n.json

```diff
@@ -241,17 +241,29 @@
         "en-tts": "Keyboard layout submenu"
     },
     "mainmenu.battery_disconnect": {
-        "en": "Disconnect battery",
-        "ja": "バッテリーを外します",
-        "zh": "断开电池",
-        "en-tts": "Disconnect battery"
+        "en": "Shutdown",
+        "ja": "シャットダウン",
+        "zh": "关闭",
+        "en-tts": "Shutdown"
+    },
+    "mainmenu.shutdown_confirm": {
+        "en": "Confirm shutdown:\nDevice will reamin off until a charging cable is plugged in. Proceed?",
+        "ja": "シャットダウンの確認:\n充電ケーブルが差し込まれるまで、デバイスはオフのままになります。続行しますか?",
+        "zh": "确认关机：\n设备将保持关闭状态，直到插入充电线。继续？",
+        "en-tts": "Confirm shutdown:\nDevice will reamin off until a charging cable is plugged in. Proceed?"
     },
     "mainmenu.reboot": {
         "en": "Reboot",
         "ja": "リブート",
         "zh": "重启",
         "en-tts": "Reboot"
     },
+    "mainmenu.lockdevice": {
+        "en": "Lock device",

```

Comment:
Maybe "Reboot and lock" sounds better? This PR removes the "Reboot" option anywhere from status menu... Users might get confused?

Reply:
I thought about this and decided maybe this was too confusing for users to have reboot & lock (because what if I wanted to just reboot?)

Instead, "disconnect battery" has been renamed to "shutdown", which I think is maybe what people would reach for in case of a reboot and that's also acceptable?

Not sure -- very open to the "Reboot and Lock" naming idea, just, worried it'll cause more confusion than it solves because it begs the question of "why reboot" during a lock without offering any explanation.